dts/bindings/usb/st,stm32-usb.yaml

```
@@ -31,5 +31,12 @@ properties:
     type: phandle
     description: PHY provider specifier
 
+  physel:
+    type: string
+    default: "USB_OTG_FS_EMBEDDED_PHY"
+    description: |
+      OTG_FS supports the following PHY interfaces:
```

The correct term for the USB controller  is `USB` or `USB_DRD_FS`, not `USB_OTG_FS`. Please update the description accordingly.
```suggestion
      USB supports the following PHY interfaces:
```